x-pack/plugins/alerting/server/routes/update_rule.ts

```diff
@@ -84,7 +92,10 @@ const rewriteBodyRes: RewriteResponseCase<PartialRule<RuleTypeParams>> = ({
   lastRun,
   nextRun,
   ...rest
-}) => ({
+}: PartialRule<RuleTypeParams>): Omit<
+  AsApiContract<PartialRule<RuleTypeParams> & { actions?: RuleResponseV1['actions'] }>,
```

Comment:
change to the latest and now versioning for now
```suggestion
  AsApiContract<PartialRule<RuleTypeParams> & { actions?: RuleResponse['actions'] }>,
```